docs/audio.rst

```diff
@@ -3,8 +3,11 @@ Audio
 
 .. py:module:: audio
```

Comment:
We also need to explain that in V2 `audio` is also part of the `microbit` module.

Reply:
Done in https://github.com/bbcmicrobit/micropython/pull/692/commits/01331f3e9d0e8ea461e9fae271fe3eb62153802f